sklearnex/linear_model/logistic_regression.py

```diff
@@ -208,12 +208,11 @@ def _onedal_gpu_fit_supported(self, method_name, *data):
                     ),
                     (self.class_weight is None, "Class weight is not supported"),
                     (self.solver == "newton-cg", "Only newton-cg solver is supported."),
+                    (self.warm_start == False, "Warm start is not supported."),
                     (
-                        self.multi_class != "multinomial",
-                        "multi_class parameter is not supported.",
+                        self.l1_ratio is None or self.l1_ratio == 0,
```

Comment:
If you really wanted to be slick:
```suggestion
                        not self.l1_ratio,
```